### ISSUE TITLE
App won't launch on Pixel 5a

### ISSUE THREAD

**Issue by GreatHornedOwl** [sparcules/Spark_Pixels#5]
Still using this great application/code for a few of my art projects!

I had previously run SparkPixels on my Pixel 5a for the past two years, but had not opened it since maybe January of this year. Now, when try to launch the app, it only says "App isn't installed." I tried reinstalling, but with the same result.

Sorry if this is not the right place to report this, I didn't know how else to contact you.

**Comment by sparcules** [comment]
@GreatHornedOwl, try clearing the app cache for Google Play Services and Google Play Store app. See the first two steps in this video: https://www.youtube.com/watch?v=ZYx9NEi1t2c&t=5s
Let me know if this works or not. Alternatively, you can email me at sparcules@gmail.com

**Comment by GreatHornedOwl** [comment]
Thanks for your quick reply. I wasn't able to resolve this with the steps you provided, but oddly I was able to launch the app from the app manager only. If you feel like this is an Android problem, I don't want to waste your time. I'll email you separately with some screenshots and let's see what you think.

**Comment by GreatHornedOwl** [comment]
Updating that the version of the app I was using did not have a digitally signed APK, and has since been updated by Kevin. I ran into a different problem after that where I could not log into Particle. This was because I had two factor authentication turned on. Going into my account via desktop browser and disabling that resolved the issue. 

Thanks to Kevin for all of his assistance for this very cool app!
